spark/v3.5/spark-runtime/LICENSE

```diff
@@ -456,95 +380,49 @@ License text:
-| AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-| LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-| OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-| THE SOFTWARE. 
-
---------------------------------------------------------------------------------
-
-This binary artifact contains Caffeine by Ben Manes.
-
+Group: com.github.ben-manes.caffeine Name: caffeine Version: 2.9.3
 Copyright: 2014-2019 Ben Manes and contributors
 Home page: https://github.com/ben-manes/caffeine
 License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Apache Arrow.
-
-Copyright: 2016-2019 The Apache Software Foundation.
+Group: org.apache.arrow Name: arrow-vector Version: 15.0.2
+Copyright: 2016-2024 The Apache Software Foundation
 Home page: https://arrow.apache.org/
 License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Netty's buffer library.
-
-Copyright: 2014-2020 The Netty Project
-Home page: https://netty.io/
-License: http://www.apache.org/licenses/LICENSE-2.0
-
---------------------------------------------------------------------------------
-
-This binary artifact contains Google FlatBuffers.
-
-Copyright: 2013-2020 Google Inc.
-Home page: https://google.github.io/flatbuffers/
+Group: org.apache.arrow Name: arrow-format Version: 15.0.2
+Copyright: 2016-2024 The Apache Software Foundation
+Home page: https://arrow.apache.org/
 License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Carrot Search Labs HPPC.
-
-Copyright: 2002-2019 Carrot Search s.c.
-Home page: http://labs.carrotsearch.com/hppc.html
+Group: org.apache.arrow Name: arrow-memory-core Version: 15.0.2
+Copyright: 2016-2024 The Apache Software Foundation
+Home page: https://arrow.apache.org/
 License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains code from Apache Lucene via Carrot Search HPPC.
-
-Copyright: 2011-2020 The Apache Software Foundation.
-Home page: https://lucene.apache.org/
+Group: io.netty Name: netty Version: 4.1.96.Final
+Copyright: 2014 The Netty Project
+Home page: https://netty.io/
 License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Apache Yetus audience annotations.
-
-Copyright: 2008-2020 The Apache Software Foundation.
-Home page: https://yetus.apache.org/
+Group: com.google.flatbuffers Name: flatbuffers-java Version: 23.5.26
+Copyright: 2013-2020 Google Inc.
+Home page: https://google.github.io/flatbuffers/
```

Comment:
Is this actually bundled in the release? I know our LICENSE mentioned it prior to this change but it looks like this is part of the test runtime.

Reply:
Yes it's in the jar (I checked the deps in the jar directly).